README.md

```diff
@@ -155,13 +155,14 @@ ESLint follows [semantic versioning](http://semver.org). However, due to the nat
 * Minor release (might break your lint build)
     * A bug fix in a rule that results in ESLint reporting more errors.
     * A new rule is created.
-    * A new option to an existing rule is created.
+    * A new option to an existing rule is created, not reporting more errors.
```

Comment:
Let's clean up the language here:

A new option to an existing rule that does not result in ESLint reporting more errors